src/site/content/en/blog/nfc/index.md

```diff
@@ -0,0 +1,906 @@
+---
+title: Interact with NFC devices on the web
+subhead: Reading and writing to NFC tags is now possible on the mobile web.
+authors:
+  - beaufortfrancois
+date: 2020-02-12
+hero: hero.jpg
+thumbnail: thumbnail.jpg
+alt: A photo of NFC tags
+description: |
+  Reading and writing to NFC tags is now possible on the mobile web.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - capabilities
+  - progressive-web-apps
+---
+
+{% Aside %}
+Web apps should be able to do anything native apps can. The [Capabilities
+project](https://developers.google.com/web/updates/capabilities), of which Web
+NFC is only a part, aims to do just that. To learn about other capabilities and
+to keep up with their progress, follow [Unlocking new capabilities for the
+web](https://developers.google.com/web/updates/capabilities).
+{% endAside %}
+
+## What is Web NFC? {: #what }
+
+Web NFC provides sites the ability to read and write to NFC tags when they are
+brought in close proximity to the user's device (usually 5-10 cm, 2-4 inches).
+The current scope is limited to NDEF, a lightweight binary message format that
+works across different tag formats.
+
+{% Aside 'gotchas' %}
+Web NFC is limited to NDEF because the security properties of reading and
+writing NDEF data are more easily quantifiable. Low-level I/O operations (e.g.
+ISO-DEP, NFC-A/B, NFC-F) and Host-based Card Emulation (HCE) are not supported.
```

Comment:
at this point?

Reply:
I want to warn developers who know already about NFC that Web NFC is NOT about payments.

Reply:
His question is more about the location than the subject. I agree. I would expect this to be with the use cases. (The title may need to change.)